### ISSUE TITLE
Climate error 0.96

### ISSUE THREAD

**Issue by Helldin** [jheling/freeathome#15]
Log Details (ERROR)
Thu Jul 18 2019 10:30:54 GMT+0200 (CEST)

Unable to prepare setup for platform freeathome.climate: Platform not found (cannot import name 'STATE_AUTO' from 'homeassistant.components.climate.const' (/usr/src/homeassistant/homeassistant/components/climate/const.py)).

**Comment by dbrouwer1979** [comment]
Same error here, after update 0.96.1 still the same error

**Comment by fotiDim** [comment]
0.96.5 works for me

**Comment by jheling** [comment]
A while ago I have asked user Lassem to take a look at it as he is the original author of this part.

**Comment by lassem** [comment]
Sorry for the super late response on this.

There's a PR up now, hopefully that will in the very least make it somewhat better.

**Comment by jheling** [comment]
Hello Lasse,

Sometimes it takes a bit longer, but it is summer, there are other things
to enjoy.
I have committed the pull request.
Thank you for the work!

Best, Johan


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virusvrij.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Op vr 23 aug. 2019 om 22:52 schreef Lasse Magnussen <
notifications@github.com>:

**Comment by dbrouwer1979** [comment]
Hi Lasse, Thank you for the updated version. I have it installed. To get my thermostat working (as it was off now) I had to turn it on first. I did this by adding the thermostat to my batches and then I could set operating mode to Heat/cool and after that it works fine (also turning it off and on again)
![Knipsel](https://user-images.githubusercontent.com/31338244/63637960-ea45c580-c682-11e9-9331-0f83cdd31f6a.PNG)
